.github/workflows/test_examples.yml

```diff
@@ -0,0 +1,35 @@
+name: Test Jupyter Notebook
```

Comment:
I would add some comments to the top of the workflow because I've been noticed that since our CI is growing I am losing track of what does what and where it is applicable. I think it can also be helpful for future developers as well so they can easily understand how to work with it